content/docs/configuration/vault.md

```diff
@@ -254,6 +256,101 @@ Kubernetes 1.24 and above.
               key: token
     ```
 
+### Authenticating with a Kubernetes Service Account Without A Secret
+
+The [previous method](#static-service-account-token) of authenticating with
+Vault has a major disadvantage: it relies on a less secure "static" token (by
+"static", we mean a token that does not have an expiry time).
```

Comment:
Don't start by dismissing the previous method. Why not just put this at the top, as the recommended method, and  start by explaining its advantages.

Reply:
I move the section at the top. Or rather, I moved it right before the existing service account authentication.